cookbook/docs/getting_started/flyte_fundamentals.md

```diff
@@ -0,0 +1,54 @@
+(getting_started_fundamentals)=
+
+# Flyte Fundamentals
+
+This section of the **Getting Started** documentation will take you through the
+fundamental concepts of Flyte: tasks, workflows, and launchplans.
```

Comment:
When mentioned as an English-language term, the Flyte entity `LaunchPlan` should be written "launch plan".
```suggestion
fundamental concepts of Flyte: tasks, workflows, and launch plans.
```